tests/cephadm/test_cluster_view.py

```diff
@@ -0,0 +1,108 @@
+from json import loads
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+DEFAULT_CEPH_DIR = "/etc/ceph/"
+DEFAULT_CONF_PATH = "/etc/ceph/ceph.conf"
+
+
+class ClusterViewError(Exception):
+    pass
+
+
+def verify_mon_service(host):
+    """
+    Verify MON service is up
+    Args:
+        host(ceph): Installer host
+    """
+    timeout, interval = 20, 6
```

Comment:
`timeout` should be multiple of `interval`, Use values 30 & 5

Reply:
Updated now.